ui/src/app/pages/analysis/notebook-redirect/component.spec.ts

```diff
@@ -9,11 +9,11 @@ import {Observable} from 'rxjs/Observable';
 
 import {NotebookComponent} from 'app/icons/notebook/component';
 import {ReminderComponent} from 'app/icons/reminder';
+import {TopBoxComponent} from 'app/pages/top-box/component';
```

Comment:
nit: I wouldn't consider a top-box to really be a page; I might put this in `components` (I think having subdirs there should be fine)

Reply:
It isn't really, but it is also going away shortly (With Jackie's PR) so I wasn't too worried about it.

Reply:
OK - sounds fine then